rest/server_context.go

```diff
@@ -284,6 +287,10 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		viewQueryTimeoutSecs = config.ViewQueryTimeoutSecs
 	}
 
+	if config.localDocExpirySecs != nil && *config.localDocExpirySecs > 0 {
```

Comment:
I think explicitly setting config.localDocExpirySecs=0 should result in no expiry.  Here it seems to be reverting to the default value.

Reply:
changed if condition to: 

 if config.localDocExpirySecs != nil && *config.localDocExpirySecs >= 0